stripe-ui-core/src/main/java/com/stripe/android/uicore/elements/OTPElementUI.kt

```diff
@@ -86,6 +117,11 @@ fun OTPElementUI(
                 modifier = Modifier
                     .weight(1f)
                     .padding(horizontal = 4.dp),
+                backgroundColor = if (enabled) {
+                    MaterialTheme.stripeColors.component
+                } else {
+                    MaterialTheme.stripeColors.placeholderText.copy(alpha = ContentAlpha.disabled)
+                },
```

Comment:
used an disabled alpha version of the placeholder text to avoid adding more color complexity.